We don't want everything in per target runtime dirs, but for openmp its required.
This is because we build openmp for amd and nvidia targets not just the native one.
--- a/openmp/CMakeLists.txt
+++ b/openmp/CMakeLists.txt
@@ -44,7 +44,7 @@
 
 # When building in tree we install the runtime according to the LLVM settings.
 # TODO: Use common runtimes infrastructure for output and install paths
-if(LLVM_ENABLE_PER_TARGET_RUNTIME_DIR AND NOT APPLE)
+if(TRUE)
   set(OPENMP_TARGET_SUBDIR "${LLVM_DEFAULT_TARGET_TRIPLE}")
   if(OPENMP_LIBDIR_SUBDIR)
     string(APPEND OPENMP_TARGET_SUBDIR "/${OPENMP_LIBDIR_SUBDIR}")
